src/components/bottom-bar/bottom-bar.tsx

```diff
@@ -0,0 +1,58 @@
+import { Component, h, Host, Prop } from '@stencil/core';
+import { defaultTheme } from '../../defaultTheme';
+import { TecnologiaTheme } from '../interfaces';
+import { ButtonPosition } from './bottom-bar.model';
+
+@Component({
+  tag: 'tec-bottom-bar',
+  styleUrl: 'bottom-bar.scss',
+  shadow: true,
+})
+export class BottomBar {
+  @Prop({ reflect: true }) theme: TecnologiaTheme = defaultTheme;
+
+  @Prop() text: string;
+
+  @Prop() buttonPosition = ButtonPosition.left;
+
+  @Prop({ mutable: false, reflect: false })
+  useAnimation = true;
```

Comment:
I tried use false to this prop, but didn't work.

https://i.imgur.com/HYznK5R.gif

Reply:
On vanilla JS to `false` values you shouldn't pass. For example, on Angular using one way data binding that will be work.

Reply:
Maybe the reflect property should resolve, i will test:

![image](https://user-images.githubusercontent.com/3449932/112625639-fb7e2780-8e0d-11eb-8155-ea02bff88c84.png)

https://stenciljs.com/docs/properties

Reply:
I tested on CodeSandbox and works. The animation class was not added to element: https://codesandbox.io/s/tecnologia-webcomponents-demo-p6tht?file=/index.html

![image](https://user-images.githubusercontent.com/3449932/112629135-a8f33a00-8e12-11eb-82f3-9401459e5e27.png)